web/app/js/components/util/Utils.js

```diff
@@ -118,12 +118,13 @@ export const numericSort = (a, b) => (_.isNil(a) ? -1 : a) - (_.isNil(b) ? -1 :
   Nicely readable names for the stat resources
 */
 export const friendlyTitle = resource => {
-  let singular = _.startCase(resource);
-  if (resource === "replicationcontroller") {
-    singular = _.startCase("replication controller");
+  let singleResource = singularResource(resource);
```

Comment:
naming nit, how about naming `singleResource` `resource` instead, and calling the function argument `inputResource` or `singularOrPluralResource` or something to make its purpose clearer? The fact that it's a single resource doesn't really matter much throughout the rest of the function?